assets/javascripts/modules/renew/renew.es6

```diff
@@ -81,20 +84,39 @@ export function send(state, errorHandler) {
         if(promoCode){
             payload.promoCode = promoCode;
         }
-            ajax({
-            type: 'json',
+        let request = new Request('/manage/renew', { 
             method: 'POST',
-            url: '/manage/renew',
-            headers: {
-                'Csrf-Token': document.querySelector('input[name="csrfToken"]').getAttribute('value')
-            },
-            contentType: 'application/json',
-            data: JSON.stringify(payload)
-        }).then((response) => {
-            window.location.assign(response.redirect);
+            body: JSON.stringify(payload),
+            headers: new Headers({
+                'Csrf-Token': document.querySelector('input[name="csrfToken"]').getAttribute('value'),
+                'Content-Type':  'application/json'
+            }),
+            credentials: 'include'
+        })
+        fetch(request).then((response) => {
+            if (response.status == 200) {
+                console.log(response)
+                response.json().then((json) => {
+                    window.location.assign(json.redirect);
+                }).catch((e) => {
+                    Raven.captureException(e)
+                    //We got a 200, and then couldn't parse the JSON.
+                    //The renewal probably worked, and we should not really be in this branch.
+                    errorHandler(renewNetworkErrorMessage)
```

Comment:
This error needs to be extremely carefully worded if we're going to return it when we receive a 200 from the API

Reply:
I think it's appropriate to show the network message. 
If we see this:
- we've changed the response
- somehow the user can't parse valid json
- the message has been changed in flight

The user should refresh then call, I think.